script.js

```diff
@@ -1,46 +1,221 @@
 const main = document.createElement("main");
-main.innerText = "hello";
 document.body.append(main);
 
 
+const monitor = document.createElement("div");
+monitor.className = "monitor";
 const screen = document.createElement("textarea");
-screen.className = "screen";
-
-main.append(screen);
 
+screen.className = "screen crt";
+monitor.append(screen);
+main.append(monitor);
 
 const keyboard = document.createElement("div");
 keyboard.className = "keyboard";
 
-main.append(keyboard); 
+main.append(keyboard);
```

Comment:
I would suggest to wrap this code into function, e.g. `setupKeyboard` or `initKeyboard`. And call it in the bottom of  the file, after all functions declarations. This function can return `keyboard`. Also it can add rows into keyboard.

Also I would suggest to wrap all your code or almost all into functions.
So your file will have the following structure:
1. config variables
2. functions
3. setup app

setup app:
```
setupKeyboard();
window.addEventListener();
window.addEventListener();
window.addEventListener();
```

Event handlers code also can be moved to separate functions